pyproject.toml

```diff
@@ -1,3 +1,45 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "geometric_kernels"
+authors = [
+    {name = "The GeometricKernels Contributors", email = "geometric-kernels@googlegroups.com"},
+]
+description="A Python Package for Geometric Kernels in TensorFlow, PyTorch and Jax"
+readme = "README.md"
+classifiers = [
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Operating System :: OS Independent",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+keywords=[
+    "geometric-kernels",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "numpy>=1.16",
+    "scipy>=1.3",
+    "plum-dispatch>=2.2.0",
+    "backends>=1.5.4",
+    "potpourri3d",
+    "robust_laplacian",
+    "opt-einsum",
+    "geomstats",
+    "einops",
+    "spherical-harmonics @ git+https://github.com/vdutor/SphericalHarmonics.git",
+]
+version="0.2a0"
```

Comment:
How about we go for version `0.2.0` instead?

Reply:
This should be read as "0.2 alpha" (as per [these guidelines](https://packaging.python.org/en/latest/discussions/versioning/#valid-version-numbers))  --- mainly for testing purposes. Once we're ready for the actual release, we'll bump the version to 0.2 (or 0.2.0, which is the same (?))